cli/module_registry_test.go

```diff
@@ -0,0 +1,37 @@
+package cli
+
+import (
+	"path/filepath"
+	"runtime"
+	"testing"
+
+	"go.viam.com/rdk/testutils"
+	"go.viam.com/rdk/testutils/inject"
+	"go.viam.com/test"
+)
+
+func TestUpdateModelsAction(t *testing.T) {
+	_, filename, _, ok := runtime.Caller(0)
+	if !ok {
+		t.FailNow()
+	}
```

Comment:
we have a helper for this and other kinds of common assertions

```suggestion
	test.That(t, ok, test.ShouldBeTrue)
```